API/CLI.md

```diff
@@ -15,6 +15,12 @@ The best place to go next is [the Tessel 2 start experience](http://tessel.io/t2
 ## Updating Tessel 2 On-board OS/Firmware
 
 ### How do I know if I need to update my T2?
+If the Tessel Module is connected to the Internet, then the CLI informs the user, which firmware version the module is currently working on. Along with this, it would also let the user know about the latest firmware version available and the command using which the user may get the newer version.
```

Comment:
This isn't actually correct! If the computer running the T2 CLI is connected to the internet when a command is run, the CLI checks whether it (the CLI) is up to date. If it is, it provides update instructions. (A further detail, maybe not necessary to show in the docs, is that it only checks once per 24 hours in order to not slow down CLI function).

Have a look:

* Here's the PR that originally merged that functionality https://github.com/tessel/t2-cli/pull/665/files
* Here's the current code (slightly more complicated, to accommodate the crash reporter) https://github.com/tessel/t2-cli/blob/master/bin/tessel-2.js#L19
* And the original issue, with link to the tooling docs for update-notifier https://github.com/tessel/t2-cli/issues/661

There is currently no automatic check for whether the connected Tessel is running up-to-date firmware, but that could be a great issue to tackle! Here it is: https://github.com/tessel/t2-cli/issues/257